database/dynamodb.go

```diff
@@ -29,34 +21,50 @@ type AWSCredentials struct {
 var db *dynamodb.DynamoDB
 
 func init() {
+	defer func() {
+		if err := recover(); err != nil {
+			fmt.Println(err)
+		}
+	}()
 
-	env, found := os.LookupEnv(Constants["ENV"])
-	if !found {
-		// load the env values using the `setUpEnv` function.
-		utils.SetUpEnv()
+	env := os.Getenv(utils.ENV)
+	if env == utils.PROD {
+		log.Println(env, " is the env")
 	}
 
 }
 
 func getAWSCredentials() *AWSCredentials {
 
+	var found bool
 	awsCredentials := new(AWSCredentials)
-	awsCredentials.Region = os.Getenv(Constants["REGION"])
-	awsCredentials.AccessKey = os.Getenv(Constants["ACCES_KEY"])
```

Comment:
code uses Constants["ACCES_KEY"] and Constants["SECRET_KEY"] as keys to retrieve environment variables. However, the correct keys should be utils.ACCESS_KEY and utils.SECRET_KEY respectively, as used in the os.LookupEnv calls. This inconsistency should be addressed.

Reply:
@iamitprakash I'm not able to see which part of the code are we talking about, can you please point us.